docs/src/architecture/architecture.md

```diff
@@ -5,10 +5,21 @@
 
 # Architecture
 
-The main architectural concept of the Ghaf platform is to break up the traditional monolithic structure to modularized components, virtual machines (VM). These VMs on hardened host OS implement the Ghaf edge virtualization platform.
+The main architectural concept of the Ghaf Platform is to break up the traditional monolithic structure to modularized components, virtual machines (VM). These VMs on hardened host OS implement the Ghaf edge virtualization platform.
 
-The Ghaf platform components are used in reference configurations to build images for reference devices:
-* [Build Configurations](../build_config/build_configurations.md)
-* [Reference Implementations](../build_config/reference_implementations.md)
+Embedded virtualization builds on cloud technologies in the development of end-to-end security. With hardware support for virtualization, we provide a hardened system of a small trusted computing base (TCB)—thin host—that enables isolation of use cases and their resources. Use cases are protected in guest virtual machines (VMs). Embedded targets are small devices (personal or headless) instead of high-performance cloud servers. Our scope is illustrated in the following diagram. For more information, see [stack](architecture/stack.md).
```

Comment:
This is nice. 
A minor formatting comment: we may want to use to use `—` (m-dash) with `thin host` and separate it with some space. Like "(TCB) — thin host — " to indicate "extra information or break in a sentence". Now it's a bit clumsy to read as the sentence looks stuffed.

Reply:
In U.S. English, the em-dash is used with no spaces on either side. For more detailed information, check the Style Guide https://github.com/tiiuae/ghaf/blob/main/docs/style_guide.md#spelling-and-punctuation.